googlemock/include/gmock/gmock-matchers.h

```diff
@@ -515,7 +515,7 @@ template <typename T, typename M>
 class MatcherCastImpl {
  public:
   static Matcher<T> Cast(const M& polymorphic_matcher_or_value) {
-    // M can be a polymorhic matcher, in which case we want to use
+    // M can be a polymorphic matcher, in which case we want to use
```

Comment:
This is not an MSVC build fix.
Please narrow your changes down for review.

If you want to fix spelling errors, do it separately.